styleguide/_static/issue-templates/documentation.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 📚 Documentation 
+about: Add/Improve Documentation
+title: 📚 <Insert Title>
+labels: documentation
+assignees: ''
+
+---
+
+# Summary
+<!---
+A short summary on what of what needs to be documented or updated.
+--->
+
+# Details
+<!---
+A detailed description about what kind of documentation should be added and/or updated and why.
+--->
+
+## Background & Context
+<!---
+Why should this documentation be added/updated?
+
+E.g.: Users are struggling to use this API and have repeatedly asked for some example code.
+--->
+
+## References
+<!---
+Further references to e.g. other information resources like links to specification(s), api docs etc.
+--->
+
+# Task(s)
+<!---
+A task list containing common task associated with this kind of issue, but also tasks specific to this issue.
+
+e.g.:
+- [ ] Update documentation of XYZ
+- [ ] Bump version number
+- [ ] Update changelog
+- [ ] ...
```

Comment:
Hm, I am thinking should we put the tasks in the issue or in the PR?

Reply:
Repeating tasks for a specific issue type also can be added to the/a PR template, still this section also can contain specific task(s) for a specific issue.